pom.xml

```diff
@@ -34,6 +34,7 @@
 		<module>org.idempiere.felix.webconsole</module>
 		<module>org.idempiere.hazelcast.service</module>
 		<module>org.idempiere.webservices</module>
+		<module>org.idempiere.webservices.helper.resoucelookup</module>
```

Comment:
I would suggest to simplify the project name to just "org.idempiere.webservices.resources"